src/plugins/plugins_ex.h

```diff
@@ -415,7 +415,7 @@ class pluginex : public plugin
 
     pluginex(drakvuf_t drakvuf, output_format_t output);
     virtual ~pluginex();
-    virtual bool stop();
+    virtual bool stop_impl() override;
```

Comment:
Why change the name from stop to stop_impl?

Reply:
I imlement common stopping logic in method `stop` of the base class `plugin` using design pattern `template method`. Derived classes must implement only specific bussiness logic in `stop_impl` pure virtual method.